nuget/Microsoft.Windows.CsWinRT.targets

```diff
@@ -147,6 +147,44 @@ $(CsWinRTFilters)
     <Copy SourceFiles="$(CsWinRTGeneratedFilesDir)\$(AssemblyName).winmd" DestinationFolder="$(TargetDir)" UseHardlinksIfPossible="false" SkipUnchangedFiles="true" />
   </Target>
 
+  <!-- ====================== -->
+  <!-- Call The GuidPatch tool on the projection .dll -->
+  <Target Name="CsWinRTInvokeGuidPatcher" Condition="'$(CsWinRTEnableGuidPatch)' == 'true'" AfterTargets="Compile" BeforeTargets="Link">
```

Comment:
I see we are doing this behind CsWinRTEnableGuidPatch being set.  What is the motivation behind that?  Is there any reason we wouldn't always do it?  Is the idea to make it opt in first, see how it goes, and then make it permanent?

Reply:
Motivated by some of our test projects tried to invoke it when they didn't need to... Let me see about making an opt out path for those projects instead of one that everyone needs to set.